### DISCUSSION TITLE
`Post.removed` returns `undefined`

### DISCUSSION THREAD

**Discussion by ghost** [thislooksfun/snoots#31] [General]
Hello again. It seems that `Post.removed` always returns `undefined` regardless of the actual state of the post. I also tried refetching, but that doesn't seem to make a difference. I'm using version dev.13.

Here's an example of a removed post: https://www.reddit.com/r/discordservers/comments/nmftww/21_cats_cat2_coffee_coffee_bread_bread/

**Comment by thislooksfun** [comment]
Yeah that sounds like a bug. Tracking progress in #32.

Btw thanks for the heads-up, but it's a lot easier for me to track if you'd make an issue next time. :)

**Reply by ghost** [reply]
Ah, sorry about that! The issue tracker directed me to discussions last time, so I did it again out of habit.

**Reply by thislooksfun** [reply]
No worries! I'm still trying to figure out how to best use discussions as well, so I get it can be a bit confusing. The rule of thumb I'm trying to go for is if I need to fix something (like this), it's an issue; otherwise it's a discussion. Basically I want to have an easy place for people to ask questions without it cluttering up the issue tracker. When in doubt just use the new issue button and it'll guide you to the right place!
